shared/logger/index.js

```diff
@@ -27,11 +27,13 @@ function _mergeSortedArraysHelper<A>(sortFn: (a: A, b: A) => number, ...arrays:
 }
 
 function deleteFileIfOlderThanMs(olderThanMs: number, filepath: string): Promise<void> {
-  return stat(filepath).then(({lastModified}) => {
-    if (Date.now() - lastModified > olderThanMs) {
-      return unlink(filepath)
-    }
-  })
+  return stat(filepath)
+    .then(({lastModified}) => {
+      if (Date.now() - lastModified > olderThanMs) {
+        return unlink(filepath)
+      }
+    })
+    .catch(() => {})
```

Comment:
else if the file isn't there we get warnings

Reply:
I noticed this earlier too, should we create the file if it's not there?